pages/agent/v3/unclustered_tokens.md

```diff
@@ -0,0 +1,143 @@
+# Unclustered agent tokens
+
+> 🚧 This page documents a deprecated feature of Buildkite
+> Previously, agents only connected to Buildkite directly via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines across to a [cluster](/docs/clusters/overview).
+> Agent tokens are now associated with clusters, such that these tokens connect to Buildkite through a specific cluster within an organization. Refer to the main [agent token](/docs/agent/v3/tokens) for details on how to create agent tokens for clusters.
```

Comment:
```suggestion
> _Agent tokens_ are now associated with clusters, such that these tokens connect to Buildkite through a specific cluster within an organization. Refer to the main [agent token](/docs/agent/v3/tokens) page for details on how to manage agent tokens for clusters.
```

Reply:
I adopted most of these great suggestions, thank you!